src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -19,9 +20,16 @@
 import java.util.Optional;
 import java.util.UUID;
 import org.apache.commons.lang3.StringUtils;
+import org.stringtemplate.v4.ST;
 
 // Common code for the ingest steps
 public final class IngestUtils {
+  private static final String INGEST_REQUEST_SCOPED_CREDENTIAL_PREFIX = "irsas_";
+  private static final String TARGET_SCOPED_CREDENTIAL_PREFIX = "dsas_";
+  private static final String INGEST_REQUEST_DATA_SOURCE_PREFIX = "irds_";
+  private static final String TARGET_DATA_SOURCE_PREFIX = "dds_";
```

Comment:
hehe Doctor of Dental Surgery (Sorry I'll leave now...)

Reply:
yeah, I'm not crazy about the prefixes. They would just be pretty long if I spell them out, but I suppose that could work.

Reply:
Also, these started with "d" because originally I was referencing the "destination" instead of the "target".